src/common/transformations/include/transformations/common_optimizations/sequence_fusion.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <openvino/pass/graph_rewrite.hpp>
+#include <transformations_visibility.hpp>
+#include <vector>
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API SequenceFusion;
+
+}  // namespace pass
+}  // namespace ov
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief SequenceFusion transformation replaces a chain of GRUCell
+ * operations with single GRUSequence op.
+ *
+ * Supported cells: GRUCell, LSTMCell, RNNCell, AUGRUCell
+ * Limitations: the source of W,R,B inputs must be the same or
```

Comment:
I don't think it is the limitation. It is the prerequisite instead.

Reply:
fixed